mat64/matrix_test.go

```diff
@@ -37,6 +38,23 @@ func panics(fn func()) (panicked bool, message string) {
 	return
 }
 
+// outside is a type that is a Matrix which does not implement other interfaces
+type outside blas64.General
```

Comment:
https://github.com/gonum/matrix/blob/master/mat64/dense_test.go#L20
https://github.com/gonum/matrix/blob/master/mat64/mul_test.go#L221

The type is yours.